src/client/components/MainForm.tsx

```diff
@@ -66,8 +67,9 @@ export const MainForm = ({
   setRecaptchaErrorMessage,
   setRecaptchaErrorContext,
   hasGuardianTerms = false,
-  onSubmitOverride,
+  onSubmit,
```

Comment:
renamed to more accurately describe what it does

Reply:
I just thought that `onSubmit` suggests that it should be the default behaviour, rather than the no javascript behaviour of the HTML form.

Reply:
I would argue that because it's an extension on top of some base JS functionality we have integrated in MainForm it makes sense to be `onSubmit` semantically as it is a callback that happens in addition to the submit behaviour of MainForm. 

With override added to it, my first thought is that it instead replaces the custom behaviour that MainForm applies and is passed directly to the underlying `form`